Tharun

tharunsnest opened the pull request.

varun1309 commented:
@tharunsnest Hi, can you update the class diagram after the changes to code? I have removed singleton pattern from most of the classes, and now classes are no longer creating instance of their own class so the diagram will look a bit different. Also added a listener package for all the component action listeners.

tharunsnest commented:
Sure, will do it.

On Mon, Sep 24, 2018, 11:45 AM Varun Srivastava <notifications@github.com>
wrote:

> @tharunsnest <https://github.com/tharunsnest> Hi, can you update the
> class diagram after the changes to code? I have removed singleton pattern
> from most of the classes, and now classes are no longer creating instance
> of their own class so the diagram will look a bit different. Also added a
> listener package for all the component action listeners.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/SER515-IllegalSkillsException/TextEditor/pull/9#issuecomment-424082572>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AgSe6tPUCJ08FCzjkOfHFf3ulUGJoIfRks5ueShJgaJpZM4W2MqA>
> .
>
